interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md

```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
 
 Specify one of the following properties to set an image:
 
-- **ImageSource**  
-    Use this property to save images along with a report definition.
-- **ImageUrl**  
-    Use this property to save only the path to the image.
+- **Image Source**  
+    Save the image to the report's definition.
+- **Image Url**  
+    Save only a path to the image.
 
 ![](../../../../../images/eurd-win-picture-box-image-property.png)
 
+## Assign an External Image
+
+Click the **Image Source** / **Image URL** property's ellipsis button and the **Open File** dialog is invoked.
```

Comment:
Change "and the **Open File** dialog is invoked" to "to invoke the **Open File** dialog"